content/kubermatic/master/upgrading/2.11_to_2.12/_index.en.md

```diff
@@ -38,3 +38,19 @@ kubectl apply -f schedules.yaml
 ```
 
 This change does not affect user cluster etcds, which are stilled backed up regularly.
+
+
+### CNI
+
+In v2.12, we are upgrading Flannel to v0.11.0 which lead to NetworkPolicies not working properly.
+Flannel doesn't detect some iptables rules and flush the appropriate rules.
+
+To avoid any failures on the cluster, you need to apply on each node of the
+user clusters the following two commands:
+
+```bash
+iptables -t nat -D POSTROUTING -s 172.25.0.0/16 ! -d 224.0.0.0/4 -j MASQUERADE
+iptables -t nat -D POSTROUTING ! -s 172.25.0.0/16 -d 172.25.0.0/16 -j MASQUERADE 
```

Comment:
the pod CIDR cannot be configured to be different from `172.25.0.0/16`? In such case we should mention that those commands should be adapted accordingly

Reply:
From Kubermatic perspective, it is hardcoded for the user clusters.
I agree though it's better to make the point clear.